qubesadmin/tests/mock_app.py

```diff
@@ -648,6 +648,12 @@ def __init__(self):
         # # this system has some reasonable defaults
         self._qubes['dom0'].default_dispvm = "default-dvm"
 
+        self._qubes['test-vm2'] = MockQube(name="test-vm2", qapp=self,
+                                           features={'menu-favorites': ''})
+        self._qubes['dom0'].features['menu-initial-page'] = 'favorites_page'
+        self._qubes['dom0'].features['menu-sort-running'] = '1'
+        self._qubes['dom0'].features['menu-position'] = 'mouse'
+
```

Comment:
I'm not against it here, but wouldn't it be more convenient to have it in desktop-linux-menu, in a wrapper?
Simply add a file somewhere in tests/ directory that does:
```python
def main():
    """
    Start the menu app
    """
    # if X is not running or other weird stuff is happening, exit with exit code
    # 6 to signal to the service that this should not be restarted
    if not Gtk.init_check()[0]:
        sys.exit(6)

    qapp = qubesadmin.tests.mock_app.MockQubesComplete()
    qapp._qubes['test-vm2'] = MockQube(name="test-vm2", qapp=self,
                                       features={'menu-favorites': ''})
    qapp._qubes['dom0'].features['menu-initial-page'] = 'favorites_page'
    qapp._qubes['dom0'].features['menu-sort-running'] = '1'
    qapp._qubes['dom0'].features['menu-position'] = 'mouse'
    dispatcher = qubesadmin.tests.mock_app.MockDispatcher(qapp)
    app = AppMenu(qapp, dispatcher)
    app.run(sys.argv)


if __name__ == '__main__':
    sys.exit(main())
```